tests/test_meshtastic_utils.py

```
@@ -88,6 +88,28 @@ def clear(self) -> None:
+    ble_future = getattr(module, "_ble_future", None)
+    if ble_future is not None:
+        done = getattr(ble_future, "done", None)
+        cancel = getattr(ble_future, "cancel", None)
+        if callable(done) and callable(cancel) and not done():
+            cancel()
+            result = getattr(ble_future, "result", None)
+            if callable(result):
+                try:
+                    result(timeout=0.2)
+                except Exception:
+                    pass
```

Ruff (0.15.6)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)